prototype_source/quantization_in_pytorch_2_0_export_tutorial.rst

```diff
@@ -0,0 +1,646 @@
+            is_dynamic=False,
+            observer_or_fake_quant_ctr=act_observer_or_fake_quant_ctr.with_args(eps=2**-12),
+        )
+
+        weight_observer_or_fake_quant_ctr: _ObserverOrFakeQuantizeConstructor = PerChannelMinMaxObserver
+        extra_args: Dict[str, Any] = {"eps": 2**-12}
+        weight_quantization_spec = QuantizationSpec(
+            dtype=torch.int8,
+            quant_min=-127,
+            quant_max=127,
+            qscheme=torch.per_channel_symmetric,
+            ch_axis=0,
+            is_dynamic=False,
+            observer_or_fake_quant_ctr=weight_observer_or_fake_quant_ctr.with_args(**extra_args),
+        )
+
+        bias_observer_or_fake_quant_ctr: _ObserverOrFakeQuantizeConstructor = PlaceholderObserver
+        bias_quantization_spec = QuantizationSpec(
+            dtype=torch.float,
+            observer_or_fake_quant_ctr=bias_observer_or_fake_quant_ctr
+        )
+        quantization_config = QuantizationConfig(
+            act_quantization_spec, weight_quantization_spec, bias_quantization_spec
+        )
+        return quantization_config
+
+2. Define the ``BackendQuantizer``
+--------------------------------------------------------
+
+Then we will define the skeleton of a ``BackendQuantizer``. The annotatation methods for each operation will be
+defined later.
+
+::
+
+    class BackendQuantizer(Quantizer):
+
+        def __init__(self):
+            super().__init__()
+            self.global_config: QuantizationConfig = None  # type: ignore[assignment]
+            self.operator_type_config: Dict[str, Optional[QuantizationConfig]] = {}
+
+        def set_global(self, quantization_config: QuantizationConfig):
+            """set global QuantizationConfig used for the backend.
+            QuantizationConfig is defined in torch/ao/quantization/_pt2e/quantizer/quantizer.py.
+            """
+            self.global_config = quantization_config
+            return self
+
+        def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
+            """annotate nodes in the graph with observer or fake quant constructors
+            to convey the desired way of quantization.
+            """
+            global_config = self.global_config
+            self.annotate_symmetric_config(model, global_config)
+
+            return model
+
+        def annotate_symmetric_config(
+            self, model: torch.fx.GraphModule, config: QuantizationConfig
+        ) -> torch.fx.GraphModule:
+            for node in reversed(model.graph.nodes):
+                # The annotation methods for each op will defined later
+                pass
+            return model
+
+        def validate(self, model: torch.fx.GraphModule) -> None:
+            """validate the annotated graph is supported by the backend"""
+            pass
+
+        @classmethod
+        def get_supported_operators(cls) -> List[OperatorConfig]:
+            """return the operator list which is supported by the backend"""
```

Comment:
Add my understanding about these questions:

- The semantic of `get_supported_operators`.
- The semantic of `OperatorConfig`.

Hi @jerryzh168, I think we need your help here for further elaboration about `get_supported_operators` since it's a  abstractmethod in base `Quantizer` which must be implemented by BackendQuantizer.

Reply:
I don't think these need to be in this doc, the main goal for this doc is to introduce different types of quantization specs and how to annotate patterns

Reply:
Got it, but we still need to define this method `get_supported_operators` for final toy example since it's an abstractmethod in base Quantizer which must be implemented by BackendQuantizer. So, did you suggest we skip this section and directly keep it in the final toy example?